### ISSUE TITLE
Spatial_oM: missing structural tee section

### ISSUE THREAD

**Issue by pawelbaran** [BHoM/BHoM#1408]
#### Description:

When working on section profile generation for Revit I noticed that we are missing the structural tee section in `BH.oM.Spatial.ShapeProfiles`:
![image](https://user-images.githubusercontent.com/26874773/178551070-50925b74-977d-4de9-a107-4379ef99ea0f.png)

@IsakNaslundBh please feel free to ignore and close if this is something that is never used in engineering 😉

**Comment by IsakNaslundBh** [comment]
You are right @pawelbaran , we only have T-profiles without tapered flanges/web.

Support for tapered flange I's and Channels where added in https://github.com/BHoM/BHoM/pull/1316 , but this was not included in that Push.
